packages/material-ui/src/ButtonBase/ButtonBase.d.ts

```diff
@@ -16,6 +16,7 @@ export interface ButtonBaseTypeMap {
     buttonRef?: React.Ref<unknown>;
     centerRipple?: boolean;
     disabled?: boolean;
+    disableFocusRipple?: boolean;
```

Comment:
That is not defined on ButtonBase but Button. See the corresponding .js files.

Reply:
Apologies, I was getting familiar with both the API philosophy and the types. It was a refactor attempt that shouldn't have made it in.